src/app/authorization/login/login.component.ts

```diff
@@ -2,52 +2,46 @@ import { Component, OnDestroy, OnInit } from '@angular/core';
 import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { Router } from '@angular/router';
 
-import { DialogService } from 'src/app/dialog-service/dialog.service';
+import { QuestionService } from '../../service/question.service';
+import { ToastService } from 'src/app/toast.service';
 import dialogData from 'src/assets/json/dialogData.json';
+
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
-  styleUrls: ['./login.component.scss'],
+  styleUrls: ['./login.component.scss']
 })
+
 export class LoginComponent implements OnInit, OnDestroy {
   userData: any;
   dialogData = { ...dialogData };
   loginForm!:FormGroup;
   
   constructor(
     private route: Router,
-    private dialogService: DialogService,
-    private fb:FormBuilder
+    private fb:FormBuilder,
+    private questionService: QuestionService,
+    public toastService: ToastService,
   ) {}
 
-  public formSubmitted(formValue:any) {
+ public formSubmitted(formValue: any) {
     let userdata = this.userData?.find(
       (value: any) => value?.email ==  formValue?.emailId && value?.password ==  formValue?.password 
     );
     if (userdata) {
+      this.toastService.showSuccessMessage('Login Successfully!');
       document.cookie = 'username' + '=' + userdata.id;
       localStorage.setItem('isAuthenticate', 'true');
       this.route.navigateByUrl('dashboard');
     } else {
-      let label = this.dialogData.loginModel.label;
-      let yesButtonLable = this.dialogData.loginModel.yesButtonLable;
-      let NoButtonLable = this.dialogData.loginModel.NoButtonLable;
-      this.dialogService
-        .openDialog(label, yesButtonLable, NoButtonLable)
-        .then((value) => {
-          if (value) {
-            this.route.navigateByUrl('registration');
-          } else {
-            this.loginForm?.reset();
-          }
-        });
+      this.toastService.showErrorMessage('Wrong Credential!');
     }
   }
 
   ngOnInit(): void {
     this.createForm();
-    if (localStorage.getItem('registeruser')?.length) {
-      let data: any = localStorage.getItem('registeruser');
+    if (localStorage.getItem('registerUser')) {
+      let data: any = localStorage.getItem('registerUser');
```

Comment:
can we manage the condition same JSON.parse(localstore.getitem('registerUser) as string)